report/web.py

```diff
@@ -328,6 +328,18 @@ def get_fixed_target(path):
       with open(os.path.join(path, name)) as f:
         fixer_prompt = f.read()
 
+      # We need to validate if the text is actually a dictionary, e.g. OpenAI
+      # prompt, as we need to beautify the string in that case.
+      try:
+        prompt_dict = json.loads(fixer_prompt)
```

Comment:
nit: structured_prompt instead? since this is not necessary a dict.

Reply:
For OpenAI, I reckon this will be a dict (but for others it might not).
Shall we beautify the string in a new function?
Erfan will push a big refactor on this file soon; having this in a standalone function will also help resolve conflicts (if any).

Reply:
Refactored into a function because we want the logic applied elsewhere as well. Fixed the naming too.